### PR TITLE
Styling improvements

### LINKED ISSUE #366: Consolidate colors
Theme colors are too complicated. Proposing the following changes:
* remove `grayXX` named colors. Add a few grays if needed, but try to solve color needs with either existing named colors or color transformations.
* review naming for simplicity.
* color adjustments to make the interface friendlier.

### PR DESCRIPTION
**Pull request by peterszerzo**
Fixes #366, #380 and #385.
* [x] consolidate theme colors.
* [x] spacing `12px` `->` `16px`
* [x] visual sweep across components

### CODE REVIEW

File: packages/components/src/Layout.tsx [hunk 1]
```diff
@@ -18,8 +18,8 @@ const Container = glamorous.div(({ theme, isSidenavExpanded }: { theme: Theme; i
   position: "relative",
   height: "100%",
   display: "grid",
-  gridTemplateColumns: `${isSidenavExpanded ? sidenavExpandedWidth : sidenavWidth}px auto`,
-  gridTemplateRows: `${headerHeight}px auto`,
+  gridTemplateColumns: `${isSidenavExpanded ? sidenavExpandedWidth : theme.unit}px auto`,
+  gridTemplateRows: `${theme.unit}px auto`,
```
Comment (TejasQ) [inline packages/components/src/Layout.tsx]: What is `theme.unit`? I'd think `unit` is something like `px`, `em`, etc...

File: packages/components/src/Paginator.tsx [hunk 2]
```diff
@@ -3,21 +3,36 @@ import * as Icon from "react-feather"
...
   className?: string
-  activeColor?: string
   disabled?: boolean
   onChange?: (page: number) => void
   maxVisible?: number
   page?: number
   pageCount: number
 }
 
+const Element = glamorous.div(
```
Comment (TejasQ) [inline packages/components/src/Paginator.tsx]: Can we name this something else? I think an `Element` is a HTML/React Element...

File: packages/components/src/Table.tsx [hunk 3]
```diff
@@ -38,12 +38,17 @@ const TableElement = glamorous.table(({ theme }: { theme: Theme }): {} => ({
...
+  },
+  "& th": {
+    padding: `${theme.spacing / 4}px ${theme.spacing}px`,
+    opacity: 0.4
+  }
+}))
+
+const TableBodyRow = glamorous.tr(({ theme, isClickable }: { theme: Theme; isClickable: boolean }): {} => ({
+  ":hover": {
+    backgroundColor: isClickable ? theme.colors.lighterBackground : "transparent"
```
Comment (TejasQ) [inline packages/components/src/Table.tsx]: Nice!

File: packages/components/src/Layout.tsx (hunk 1 above)
Comment (peterszerzo) [reply packages/components/src/Layout.tsx]: The height for a unit in the side navigation, equal to the height of the header. Any ideas for a better name?

Comment (TejasQ) [reply packages/components/src/Layout.tsx]: `item`? `itemHeight`? What is a `unit`?

**Pull request merged**